templates/samples/messageInformation/index.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Generated using theia-plugin-generator
+ */ 
+import * as theia from '@theia/plugin';
+
+const disposables: theia.Disposable[] = [];
+
+export function start() {
+    const informationCommand: theia.Command = {
+        id: 'information-message-command',
+        label: 'information-message-command',
+    };
+    const informationModalCommand: theia.Command = {
+        id: 'information-modal-message-command',
+        label: 'Information Modal Message',
+    };
+    
+    disposables.push(
+        theia.commands.registerCommand(informationCommand, async (...args: any[]) => {
+            const action = await theia.window.showInformationMessage('Information message!');
+            console.log('Close information message', 'Resolve', action);
```

Comment:
should we avoid console.log ? for now with a backend plugin we may don't see it.